py/desisim/bal.py

```diff
@@ -0,0 +1,108 @@
+"""
+desisim.bal
+===========
+
+Functions and methods for inserting BALs into QSO spectra.
+
+"""
+from __future__ import division, print_function
+
+import numpy as np
+from astropy.io import fits 
+import random 
+
+def readbaltemplates(file):
+    """ Read in the BAL templates fits file
```

Comment:
`file` used to be a built-in function (in python 2.x), although it isn't anymore (in 3.x).  Nevertheless, my EMACS buffer thinks it's reserved(!), so to avoid confusion you might consider using a different name (e.g., `balfile`).

Reply:
Done